airflow/providers/google/marketing_platform/operators/display_video.py

```diff
@@ -635,6 +635,10 @@ def execute(self, context: dict) -> Dict[str, Any]:
         self.log.info("Creating operation for SDF download task...")
         operation = hook.create_sdf_download_operation(body_request=self.body_request)
 
+        name = operation["name"]
+        self.xcom_push(context, key="name", value=name)
+        self.log.info("Created SDF operation with name: %s", name)
```

Comment:
Nice! This will definitely simplify accessing the value - currently users still can access it doing via `{{ task_instance.xcom_pull("task_id")["name"] }}`